packages/matter.js-react-native/src/net/UdpChannelReactNative.ts

```diff
@@ -8,11 +8,49 @@ import dgram from "react-native-udp";
 import { Diagnostic, Logger } from "@project-chip/matter.js/log";
 import { MAX_UDP_MESSAGE_SIZE, NetworkError, UdpChannel, UdpChannelOptions } from "@project-chip/matter.js/net";
 import { ByteArray } from "@project-chip/matter.js/util";
-import type { RemoteInfo, Socket, SocketOptions } from "dgram";
```

Comment:
This is a node import. Probably rn crypto or one of its dependencies has an improper dependency on node's types that they fixed so it's no longer available to you unless you add it to tsconfig.

Given that a.) these are intended to be drop-in replacements for node libs and b.) you'll need to use a single type definition to have a shared implementation...  I would just cast the top-level imports to node's types and be done with it.

So e.g. something like:

```typescript
const dgram = rnDgram as unknown as typeof global.dgram;
const crypto = rnCrypto as unknown as typeof global.crypto;
```

Reply:
I add this comment as link to the cleanup todo.